src/grpc/keygen/result/multisig.rs

```diff
@@ -106,9 +107,15 @@ impl Service {
                         keygen_init.my_index
                     ));
                 }
-                // TODO: get `.pub_key_bytes()`, do basic validity checks and return (public keys, recovery info, secret_key_shares)
-                let output_bytes = serialize(&secret_key_shares[0].group().all_verifying_keys())
-                    .map_err(|_| anyhow!("Cannot serialize multisig output"))?;
+                // TODO: do basic validity checks and return (public keys, recovery info, secret_key_shares)
+                // TODO: change the protobuf to send a Vec<KeygenShare>?
```

Comment:
Yes, we'll need to change the protobuf.  Let's coordinate with axelar-core peeps on this.  The purpose of `all_encoded_pubkeys` is to give you something specifically intended for grpc, so we have the freedom to change it to suit our needs.  (eg. not sure yet whether axelar-core wants `party_id`, `subshare_id`).